dev/com.ibm.ws.cdi.extension_fat/fat/src/com/ibm/ws/cdi/extension/apps/invocationContext/InvocationContextTestServlet.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.extension.apps.invocationContext;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@SuppressWarnings("serial")
+@WebServlet("/InvocationContextTestServlet")
+public class InvocationContextTestServlet extends FATServlet {
+
+    @Inject
+    InterceptedBean ib;
+
+    @Test
+    public void testGetInterceptorBindingsFromInvocationContext() {
+        ib.iExist();
+    }
```

Comment:
The asserts are in the interceptor, which makes sense, but this test method needs to ensure that the interceptor is actually called.

You could make the interceptor change the return value and have this method check for the expected value.

Also add a comment here to say that the asserts are in the interceptor, otherwise it's hard to see what the test is doing.